### PR TITLE
Frontend 👓

### PR DESCRIPTION
**Pull request by teocomi**
Alpha frontend, addresses most of #40

### CODE REVIEW

**Comment by didimitrie** [comment]
The frontend fails to build on a fresh clone: 
```
 ERROR  Failed to compile with 1 errors                 12:00:53 PM

This dependency was not found:

* v-tooltip in ./src/main-frontend.js

To install it, you can run: npm install --save v-tooltip
```

Also, as a quick question, why was a separate [tooltip library](https://github.com/Akryum/v-tooltip) needed vs the [vuetifyjs](https://vuetifyjs.com/en/components/tooltips/) one? I might be missing something!

File: frontend/src/AppFrontend.vue [hunk 1]
```diff
@@ -1,33 +1,360 @@
...
-    </v-content>
+    </v-main>
   </v-app>
 </template>
 <script>
-import HelloWorld from './components/HelloWorld';
+import userQuery from "./graphql/user.gql"
+import gql from "graphql-tag"
 
 export default {
```
Comment (didimitrie) [inline frontend/src/AppFrontend.vue:134]: Search doesn't seem to be debounced? Not a blocker but to remember this is a self-inflicted DDoS route that should be patched.

Comment (teocomi) [reply frontend/src/AppFrontend.vue:134]: Not yet implemented, would you like me to add it to this PR or make an issue for it?

**Comment by teocomi** [comment]
> Quick question, why was a separate [tooltip library](https://github.com/Akryum/v-tooltip) needed vs the [vuetifyjs](https://vuetifyjs.com/en/components/tooltips/) one? I might be missing something!

The default tooltip requires smth like 5-6 lines of code for each tooltip this lib adds a "v-tooltip" prop to any component and voila'!

**Comment by didimitrie** [comment]
> The default tooltip requires smth like 5-6 lines of code for each tooltip this lib adds a "v-tooltip" prop to any component and voila'!

feel you dude, checked the docs for the other one and i can understand the pain... otherwise, we're good to merge pending issue on debouncing (i don't want this to slip into something like a beta!)

**Pull request merged**